src/overlays/actors/ovl_En_Ds2n/z_en_ds2n.h

```diff
@@ -2,16 +2,24 @@
 #define Z_EN_DS2N_H
 
 #include "global.h"
+#include "objects/object_ds2n/object_ds2n.h"
 
 struct EnDs2n;
 
 typedef void (*EnDs2nActionFunc)(struct EnDs2n*, PlayState*);
 
 typedef struct EnDs2n {
     /* 0x0000 */ Actor actor;
```

Comment:
```suggestion
    /* 0x000 */ Actor actor;
```

Reply:
ah yeah sorry this one always slips past my vision.